Chart: Add Enhance Performance How To topic (#7492)
DevExpress/devextreme-documentation: concepts/05 UI Components/Chart/93 Enhance Performance on Large Datasets.md

> When working with large datasets, you can improve Chart performance by rendering less data. This help topic covers multiple ways to do so.

Reviewer: I believe Ray asked us not to use "cover" if we can find an alternative. In news, you "cover" an ongoing story. We mostly explain or describe how to do stuff. 

```suggestion
When working with large datasets, you can improve Chart performance by rendering less data. This help topic describes multiple ways to do so.
```

> When working with large datasets, you can improve Chart performance by rendering less data. This help topic describes multiple ways to do so.

Reviewer 2: ```suggestion
This help topic describes how to improve Chart performance by rendering less data when working with large datasets. 
```

### Implement Zoom and Pan Operations

> Enable zooming and panning to improve performance on datasets with long argument axes. These operations allow users to view the entirety of your dataset using scrolling. You can also implement on-demand data loading to improve the component's performance. For more information about loading data on-demand, refer to the following demo:

Reviewer: ```suggestion
Activate zoom and pan operations to improve performance on datasets with long argument axes. These operations allow users to display smaller data sub-sets and scroll the chart horizontally. You can also implement on-demand data loading to further optimize the component's performance. For more information about loading data on-demand, refer to the following demo:
```

> Activate zoom and pan operations to improve performance on datasets with long argument axes. These operations allow users to display smaller data sub-sets and scroll the chart horizontally. You can also implement on-demand data loading to further optimize the component's performance. For more information about loading data on-demand, refer to the following demo:

Reviewer 2: ```suggestion
Enable zoom and pan to improve performance with long argument axes. These features allow users to view smaller data sets and scroll through the chart. To optimize further, use on-demand data loading. See the demo for details:
```

#include btn-open-demo with {
    href: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/Charts/LoadDataOnDemand/"
}

### Data Aggregation

> Aggregation groups data points by averaging them. Enable data aggregation to render fewer data points. Set one of the following properties to `true`:

Reviewer: ```suggestion
You can aggregate data - group arguments and display a summary value. Use this method to render fewer data points. Set one of the following properties to `true`:
```

> You can aggregate data - group arguments and display a summary value. Use this method to render fewer data points. Set one of the following properties to `true`:

Reviewer 2: ```suggestion
You can group data and show summary values to reduce the number of points. To do this, set one of the following properties to `true`:
```

- **series**.**aggregation**.[enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/aggregation/#enabled): Enables aggregation for a specific series.
- **commonSeriesSettings**.**aggregation**.[enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/aggregation/#enabled): Enables aggregation for all series.

For discrete axes, you do not need to define additional parameters to enable aggregation. For continuous and logarithmic axes, specify one of the following properties:

> - **argumentAxis**.[aggregationInterval](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/aggregationInterval/): Defines aggregation interval in axis units.

Reviewer 2: ```suggestion
- **argumentAxis**.[aggregationInterval](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/aggregationInterval/): Defines the aggregation interval in axis units.
```

> - **argumentAxis**.[aggregationGroupWidth](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/#aggregationGroupWidth): Defines aggregation interval in pixels.

Reviewer 2: ```suggestion
- **argumentAxis**.[aggregationGroupWidth](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/#aggregationGroupWidth): Defines the aggregation interval in pixels.
```

[note] If you specify both properties, **aggregationInterval** overrides **aggregationGroupWidth**.

For more information on data aggregation, refer to the following help topic: [Data Aggregation](/Documentation/Guide/UI_Components/Chart/Data_Aggregation/).

### Hide Unnecessary Chart Elements

> You can hide unnecessary elements to improve Chart performance. For instance, disable **series**.**point**.[visible](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/point/#visible) to render only chart lines. You can also hide unused Chart elements and controls, as well as disable animations.

Reviewer: ```suggestion
You can hide unnecessary elements to improve Chart performance. For instance, disable **series**.**point**.[visible](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/point/#visible) to render only chart lines, without markers. You can also hide unused Chart elements and controls or disable animations.
```

> You can hide unnecessary elements to improve Chart performance. For instance, disable **series**.**point**.[visible](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/point/#visible) to render only chart lines, without markers. You can also hide unused Chart elements and controls or disable animations.

Reviewer 2: ```suggestion
You can hide unnecessary elements to improve Chart performance. For instance, disable **series**.**point**.[visible](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/point/#visible) to render chart lines without markers. You can also hide unused Chart elements and controls or disable animations.
```
